src/shapes/object.class.js

```diff
@@ -592,7 +592,7 @@
       'top left width height scaleX scaleY flipX flipY originX originY transformMatrix ' +
       'stroke strokeWidth strokeDashArray strokeLineCap strokeDashOffset strokeLineJoin strokeMiterLimit ' +
       'angle opacity fill globalCompositeOperation shadow visible backgroundColor ' +
-      'skewX skewY fillRule paintFirst clipPath strokeUniform'
+      'skewX skewY fillRule paintFirst clipPath erasable strokeUniform'
```

Comment:
erasable isn't a state property. A state property is meant to cause a re-render or fire a modified when changed in stateful mode. I do not think erasable is useful in that case.

Reply:
DONE